libs/auth-api-lib/src/lib/resources/dto/domain.dto.ts

```diff
@@ -22,4 +22,18 @@ export class DomainDTO {
     example: '0123456789',
   })
   readonly nationalId!: string
+
+  @IsString()
+  @IsNotEmpty()
+  @ApiProperty({
+    example: 'Mínar síður Ísland.is',
+  })
+  readonly displayName!: string
+
+  @IsString()
+  @IsNotEmpty()
+  @ApiProperty({
+    example: 'Stafrænt Ísland',
+  })
```

Comment:
Maybe use the description field to add a description that this key is used for to look up the organisation in Contentful.